### ISSUE TITLE
typedef-whitespace onespace doesn't accept newlines

### ISSUE THREAD

**Issue by spiffytech** [palantir/tslint#1521]
### Bug Report

- __TSLint version__:  3.15.1
- __TypeScript version__:  2.1.0-dev.20160731
- __Running TSLint via__:  CLI / vim syntastic

#### TypeScript code being linted

```ts
/**
 * Problem comes up when function signature exceeds line length limit
 * and I have to break it onto multiple lines
 */
function example(
  {a, b}:
  {a: number, b: number}
) {
  return "Hello, World!";
}
```

with `tslint.json` configuration:

```json
"extends": "tslint:recommended"
```

#### Actual behavior

test.ts[2, 10]: expected onespace after colon in parameter

#### Expected behavior

tslint should accept a newline as a valid separator for type definitions

**Comment by adidahiya** [comment]
fixed by #1531

**Comment by spiffytech** [comment]
Can you please publish a bugfix release that includes this fix?

**Comment by adidahiya** [comment]
4.0.0-dev.0 and 4.0.0-dev.1 include the fix, does that work for you? or do you need a 3.15 patch

**Comment by spiffytech** [comment]
How many more breaking changes are expected before 4.0 final? If most breaking changes are already implemented, I could upgrade to dev.1. But if more are coming, I'd prefer a 3.15 patch so I can get this fix now, and then do all of my 4.0 changes later, in a single batch.
